pages/linux/mcookie.md

```diff
@@ -0,0 +1,19 @@
+# mcookie
+
+> Generate a 128-bit random hexadecimal number.
```

Comment:
Perhaps this will read better?
```suggestion
> Generates random 128 bit numbers in hexadecimal.
```

Reply:
@sbrl What about this ?

Generates random 128 bit hexadecimal number.

Reply:
Almost, @nsuthison. You're missing an article there.
```suggestion
> Generates random 128 bit hexadecimal numbers.
```
It either has to be plural (`numbers`), or include an article (`a random number`).